### PR TITLE
Add blog post, "Introducing the Catalyst Project..."

### PR DESCRIPTION
**Pull request by NPDebs**
:+1::tada: First of all, thanks for taking the time to contribute! :tada::+1:

## FOR CONTRIBUTOR

* [x] I have read the [CONTRIBUTING.md](https://github.com/open-life-science/open-life-science.github.io/blob/main/CONTRIBUTING.md) document

PR Type: 
* [x] This PR adds a new blog post
* [ ] This PR does something else (explain above)

 
## FOR REVIEWERS

Thanks for taking the time to review! :heart:

Here are the list of things to make sure of:
* [x] The website builds (a check will fail if not)
* [x] All images have been added within the Pull Request and they have Alt text
* [x] If there are paragraphs or text, the key messages are highlighted
* [x] All internal links (within OLS website) use the [`{% link path_to_file.md %}` format](https://jekyllrb.com/docs/liquid/tags/#link)
* [x] The preview corresponds to the changes described in the Pull Request
* [x] The code is tidy and passes the linting tests

### CODE REVIEW

File: 2024-10-31-introducing-the-catalyst-project-community-partner-highlights.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title: Introducing the Catalyst Project Community Partner Highlights
+authors: 
+- tajuddeen1
```
Comment (NPDebs) [inline 2024-10-31-introducing-the-catalyst-project-community-partner-highlights.md:5]:
```suggestion
- slldec
```

File: 2024-10-31-introducing-the-catalyst-project-community-partner-highlights.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title: Introducing the Catalyst Project Community Partner Highlights
+authors: 
+- slldec
```
Comment (NPDebs) [inline 2024-10-31-introducing-the-catalyst-project-community-partner-highlights.md:5]:
```suggestion
- katiepratt
- slldec
```

File: 2024-10-31-introducing-the-catalyst-project-community-partner-highlights.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
+---
+layout: post
+title: Introducing the Catalyst Project Community Partner Highlights
+authors: 
+- katiepratt
+- slldec
+image: https://images.unsplash.com/photo-1581093577421-f561a654a353
```
Comment (NPDebs) [inline 2024-10-31-introducing-the-catalyst-project-community-partner-highlights.md:7]:
```suggestion
image: https://media.licdn.com/dms/image/v2/D5610AQGPZaXMJKu5Wg/image-shrink_800/image-shrink_800/0/1730385031498
```

**Comment by NPDebs** [comment]
@yochannah - We've reviewed the PR and are both happy with it from the technical and aesthetic aspects. Please approve for merge after review. Thanks! ⭐

File: 2024-10-31-introducing-the-catalyst-project-community-partner-highlights.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
+---
+layout: post
+title: Introducing the Catalyst Project Community Partner Highlights
+authors: 
+- Katie Pratt
+- slldec
+image: https://media.licdn.com/dms/image/v2/D5610AQGPZaXMJKu5Wg/image-shrink_800/image-shrink_800/0/1730385031498?e=1731063600&v=beta&t=-F0ZI4Zj-oKi2cRQyrZcypICq2RuwQAaNyJQubPH3mg
+photos:
+  name: Picture by thisisengineering
```
Comment (NPDebs) [inline 2024-10-31-introducing-the-catalyst-project-community-partner-highlights.md:9]:
```suggestion
  name: Picture by The Catalyst Project
```

File: 2024-10-31-introducing-the-catalyst-project-community-partner-highlights.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+layout: post
+title: Introducing the Catalyst Project Community Partner Highlights
+authors: 
+- Katie Pratt
+- slldec
+image: https://media.licdn.com/dms/image/v2/D5610AQGPZaXMJKu5Wg/image-shrink_800/image-shrink_800/0/1730385031498?e=1731063600&v=beta&t=-F0ZI4Zj-oKi2cRQyrZcypICq2RuwQAaNyJQubPH3mg
+photos:
+  name: Picture by The Catalyst Project
+  license: CC BY-SA 4.0
+  url: https://unsplash.com/@thisisengineering
```
Comment (NPDebs) [inline 2024-10-31-introducing-the-catalyst-project-community-partner-highlights.md:11]: https://www.linkedin.com/posts/2i2c-org_as-part-of-our-work-on-the-catalyst-project-activity-7257760964647755777-4fNn

Comment (NPDebs) [inline 2024-10-31-introducing-the-catalyst-project-community-partner-highlights.md:11]:
```suggestion
  url: https://www.linkedin.com/posts/2i2c-org_as-part-of-our-work-on-the-catalyst-project-activity-7257760964647755777-4fNn
```

**Comment by yochannah** [comment]
@NPDebs if you're happy with the technical parts, please go through the review checklist at the top, then go ahead and merge!

**Comment by NPDebs** [comment]
I'm happy with the technical changes, and I've gone through the checklist.

But I wouldn't want to merge my own PR. 🌝

**Review by yochannah — APPROVED** [review]

**Pull request merged**